src/Cake.NuGet.Tests/Stubs/FakeNuGetSettingSection.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using NuGet.Configuration;
+
+namespace Cake.NuGet.Tests.Stubs
+{
+    internal sealed class FakeNuGetSettingSection : SettingSection
+    {
+        public FakeNuGetSettingSection(string name, IReadOnlyDictionary<string, string> attributes, IEnumerable<SettingItem> children)
+            : base(name, attributes, children)
+        {
+        }
+
+        public void AddItem(SettingItem item) => Children.Add(item);
```

Comment:
Don't use expression-bodied methods. We're not using it anywhere else in the code base.